gadget-container/entrypoint.sh

```diff
@@ -41,6 +41,18 @@ echo $GADGET_IMAGE
 
 echo "Gadget image flavour: ${GADGET_IMAGE_FLAVOUR}"
 
+echo -n "Namespaces settings: "
+if [ "$(stat -L -c %i /proc/self/ns/pid)" = "$(stat -L -c %i /host/proc/1/ns/pid)" ] ; then
+  echo -n "hostPID=true "
+else
+  echo -n "hostPID=false "
+fi
+if [ "$(stat -L -c %i /proc/self/ns/net)" = "$(stat -L -c %i /host/proc/1/ns/net)" ] ; then
+  echo "hostNetwork=true"
+else
+  echo "hostNetwork=false"
+fi
+
```

Comment:
I moved it in `gadget-container/gadgettracermanager/main.go` but somehow it breaks `kubectl-gadget`. Notice that the first two lines are not valid json:
```
$ ./kubectl-gadget advise network-policy monitor --output -
INFO[0000] HostPID=false                                
INFO[0000] HostNetwork=false                            
{"node":"minikube-docker","type":"debug","message":"tracer attached","uid":0,"gid":0,"dst":{"kind":"raw"}}
{"node":"minikube-docker","type":"debug","message":"tracer attached","uid":0,"gid":0,"dst":{"kind":"raw"}}
```

Other gadgets are not impacted.

Reply:
This is because the "advise network-policy" gadget still use the kubectl-exec API and it executes `gadgettracermanager -receive-stream`... I'm fixing it :)

Reply:
I faced a similar issue in https://github.com/inspektor-gadget/inspektor-gadget/pull/1743 and fixed it by moving those lines to be only printed when `serve` is true -> https://github.com/inspektor-gadget/inspektor-gadget/commit/da18f5b36b0d844120e9da76a6fcdf3b254a26d2